scripts/bash/enceladus_env.template.sh

```diff
@@ -56,11 +52,26 @@ DEFAULT_DEPLOY_MODE="client"
 
 LOG_DIR="/tmp"
 
+# Kafka security
+# Path to jaas.config
+#JAAS_CLIENT="-Djava.security.auth.login.config=/path/jaas.config"
+#JAAS_CLUSTER="-Djava.security.auth.login.config=jaas_cluster.config"
+
+APPLICATION_PROPERTIES_CLIENT="-Dconfig.file=/absolute/path/application.conf"
+APPLICATION_PROPERTIES_CLUSTER="-Dconfig.file=application.conf"
+
+# Files to send when running in cluster mode (comma separated)
+ENCELADUS_FILES="/absolute/path/application.conf#application.conf"
```

Comment:
Here is the file from where Menas URIs will be taken.

Reply:
It's a file 'alias': https://stackoverflow.com/a/49866757/1038282

Reply:
Need trick 👍 Maybe a comment explaining?